velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -649,6 +649,16 @@ class PlanBuilder {
       const std::vector<std::string>& unnestColumns,
       const std::optional<std::string>& ordinalColumn = std::nullopt);
 
+  /// Add a WindowNode to execute a list of windowFunctions.
+  /// @param windowFunctions A list of one or more window function SQL like
+  /// strings to be computed by this windowNode.
+  /// A window function SQL string looks like :
```

Comment:
Perhaps clarify that PARTITION BY, ORDER BY and frame clauses are optional. Maybe include a few examples.